Subject:  audit: export symbol for use with xt_AUDIT
From: Jan Engelhardt <jengelh@medozas.de>
Patch-mainline: 2.6.39
Git-commit: ae9d67aff60af59548b6c7d1a74febea09660122
Acked-by: Jiri Bohac <jbohac@suse.cz>
References: fate#312390, fate#312386

When xt_AUDIT is built as a module, modpost reports a problem.

MODPOST 322 modules
ERROR: "audit_enabled" [net/netfilter/x_tables.ko] undefined!
WARNING: modpost: Found 1 section mismatch(es).

Cc: Thomas Graf <tgraf@redhat.com>
Signed-off-by: Jan Engelhardt <jengelh@medozas.de>

diff --git a/kernel/audit.c b/kernel/audit.c
index 77770a0..5842f65 100644
--- a/kernel/audit.c
+++ b/kernel/audit.c
@@ -74,6 +74,8 @@ static int	audit_initialized;
 int		audit_enabled;
 int		audit_ever_enabled;
 
+EXPORT_SYMBOL_GPL(audit_enabled);
+
 /* Default state when kernel boots without any parameters. */
 static int	audit_default;
 
